common/ECSResourcesCheck/Jenkinsfile

```diff
@@ -26,10 +24,15 @@ pipeline {
 
     stage('Compare ECS clusters') {
       steps {
-      sh '''cd common/ECSResourcesCheck/target
-            java -jar ECSClusterComparison-1.0-SNAPSHOT-jar-with-dependencies.jar ${Cluster} ${aws_access_key_id} ${aws_secret_access_key} ${region} "${baseJson}"
-         '''
-         }
-
+        withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'aws_folio_int', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]){
```

Comment:
Right now it's ok to hard-code it as such, but if you have time please parameterize it so that the account ID ("PRD" | "INT" | "devQA")  can be selected and passed in from the form.